examples/gke-daemonset-raid-disks.yaml

```diff
@@ -0,0 +1,67 @@
+# 1. Run gcloud CLI with --local-nvme-ssd-block option to create a node pool with
```

Comment:
Can you also update https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/getting-started.md#option-2-gke?

Reply:
Done